devtools/ci/sync-mainnet.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
```

Comment:
Could we add these added features to the existing [sync-mainnet.sh](https://github.com/nervosnetwork/ckb-integration-test/blob/main/ckb-sync-mainnet/script/sync-mainnet.sh) but now create a new script?

Reply:
done

Reply:
[sync-mainnet.sh](https://github.com/nervosnetwork/ckb-integration-test/blob/main/ckb-sync-mainnet/script/sync-mainnet.sh) already integrate this function. This file should be deleted.

Reply:
done